bentoml/_internal/configuration/containers.py

```diff
@@ -410,6 +427,29 @@ def tracer_provider(
                 agent_host_name=jaeger_server_address,
                 agent_port=jaeger_server_port,
             )
+            provider.add_span_processor(BatchSpanProcessor(exporter))  # type: ignore (no opentelemetry types)
+            return provider
+        elif (
+            tracer_type == "otlp"
+            and otlp_server_protocol is not None
+            and otlp_server_url is not None
+        ):
+            if otlp_server_protocol == "grpc":
+                # pylint: disable=no-name-in-module # https://github.com/open-telemetry/opentelemetry-python-contrib/issues/290
```

Comment:
If you want to put pylint disable you probably ned to put it inline with L439

Reply:
Thanks for the review! I don't have a problem modifying this but to be honest I re-used the same hints used for zipkin and jaeger from lines 406 and 421. Maybe @sauyon can chime in since these hints were originally added in their PR: https://github.com/bentoml/BentoML/pull/2632 .